modules/sharedtest/testobjects/alpine-vm.go

```diff
@@ -1,6 +1,9 @@
 package testobjects
 
-import v1 "kubevirt.io/api/core/v1"
+import (
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/kubevirt/pkg/pointer"
```

Comment:
I don't think we should be adding another dependency on `kubevirt.io/kubevirt` here. Importing this package in another project is asking for trouble.

Can you use the k8s pointer package instead?

/hold

Reply:
Thanks! I saw that pointer package is deprecated and it suggested to use `k8s.io/utils/ptr` instead. I replaced it with it.